packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -100,7 +102,8 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.lookback,
       getTime,
       config.invertPrice, // Not checked in config because this parameter just defaults to false.
-      config.priceFeedDecimals // This defaults to 18 unless supplied by user
+      config.priceFeedDecimals, // This defaults to 18 unless supplied by user
+      uniswapBlockCache
```

Comment:
how does this play in server less?

Reply:
I think it would create a single cache per run, and each run might have multiple uniswap block caches. Are there instances in a single run where the same UniswapPriceFeed is updated twice?